mangaki/mangaki/utils/anidb.py

```diff
@@ -132,6 +132,34 @@ def _build_work_titles(self,
 
         return missing_titles
 
+    def get_XML(self, anidb_aid):
+        anidb_aid = int(anidb_aid)
```

Comment:
We should assert that `anidb_aid` received is always an integer rather than converting it ourselves, it may hide external bugs.

By assert, I mean, we should either:

- do nothing.
- add the [`int` type hinting](https://docs.python.org/3.5/library/typing.html)

In this example, it'd become:

```python
def get_xml(self, anidb_aid: int):
    pass
```